### ISSUE TITLE
Train  tab get stuck on Training in Progress if Compreface is down.

### ISSUE THREAD

**Issue by Chreece** [jakowenko/double-take#71]
Steps to reproduce:

- Stop compreface (by me, it needs to get restarted after reboot, even if I run it with the option -d on docker-compose)
- Select a person profile in Train tab
- Add a photo (+)

Now you have a screen like that anytime that you open Train Tab:
![Screenshot 2021-08-14 164337](https://user-images.githubusercontent.com/68458228/129450170-a2ccd822-3947-48e1-b05a-7c251e83bf8f.png)

Double-Take Version:
1.0.0-167758f:Beta

Compreface Version:
0.5.1

For any more info, happy to share :)

**Comment by Chreece** [comment]
Config:
`server:
  port: 3000

mqtt:
  host: mqtturl
  username: mqttuser(yeah sure)
  password: mqttpass(not really)
  topics:
    frigate: frigate/events
    matches: double-take/matches
    cameras: double-take/cameras

home_assistant:
  url: HAurl
  token: HAtoken

confidence:
  match: 90
  unknown: 40

objects:
  face:
    min_area_match: 10000

save:
  matches: true
  unknown: true

purge:
  matches: 168
  unknown: 10

frigate:
  url: frigateurl
  image:
    height: 1088
  attempts:
    latest: 10
    snapshot: 10
  cameras:
    - door

detectors:
  compreface:
    url: compurl
    key: mykey
    face_plugins: age,gender
  deepstack:
    url: deepsturl

time:
  format: F
  timezone: UTC+2
`

**Comment by jakowenko** [comment]
Thanks for reporting this. I was aware of this edge case happening. I'll try to get a fix out this week. Is it still stuck for you? One thing you can try to do in the meantime to get it back working is to go to the train page and train a new image for a user to force the process to pick back up.

**Comment by Chreece** [comment]
Hey David 👋 
Thank you for your response. 
I tried to add another photo, the screen now stays in "Training in progress", only the numbers have changed ;)

**Comment by Chreece** [comment]
I just saw the dimensions of the previous snapshot on my laptop. 
Sorry for that, I was on my phone :(

**Comment by jakowenko** [comment]
Hey @Chreece, can you give the new beta build a try? Upon restarts any of those orphaned untrained records should be deleted. I also catch errors related to failed trainings better and will show them on the train page. Here's an example.

<img width="485" alt="Screen Shot 2021-08-16 at 11 51 53 PM" src="https://user-images.githubusercontent.com/1081811/129660955-beb6e8a4-1485-4bdd-9670-d1120e44f6b9.png">

**Comment by Chreece** [comment]
I have downloaded the latest beta. If I will meet any difficulties I will reply. I don't want to test again the train process with compreface down, since I have already deleted all the train faces and the double-take db to get away that screen. It's a very time demanding- process to retrain all the faces and I would avoid it...

Thank you very much for your kindness, spending so much private time on a project and then sharing it with the community for free!
Keep up David!
